connectors/jdbc/src/main/java/io/camunda/connector/jdbc/model/request/connection/DetailedConnection.java

```diff
@@ -9,24 +9,37 @@
 import io.camunda.connector.generator.java.annotation.TemplateProperty;
 import io.camunda.connector.generator.java.annotation.TemplateSubType;
 import io.camunda.connector.jdbc.model.request.SupportedDatabase;
+import io.camunda.connector.jdbc.utils.ConnectionStringHelper;
 import jakarta.validation.constraints.NotBlank;
-import java.util.Optional;
-import java.util.function.Function;
+import java.util.Map;
+import java.util.Properties;
 
 @TemplateSubType(id = "detailed", label = "Detailed")
 public record DetailedConnection(
     @NotBlank @TemplateProperty(group = "connection", label = "Host") String host,
     @NotBlank @TemplateProperty(group = "connection", label = "Port") String port,
     @TemplateProperty(group = "connection", label = "Username") String username,
-    @TemplateProperty(group = "connection", label = "Password") String password)
+    @TemplateProperty(group = "connection", label = "Password") String password,
+    @TemplateProperty(group = "connection", label = "Database name") String databaseName,
+    @TemplateProperty(
+            group = "connection",
+            label = "Properties",
+            description =
+                "Provide the payload for the event as JSON. Details in the <a href=\"https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/amazon-eventbridge/?awseventbridge=outbound\" target=\"_blank\">documentation</a>") // TODO description
```

Comment:
I believe, the link to doc is incorrect

Reply:
indeed, there will a documentation PR and then I'll update the code accordingly :)